app/src/main/java/org/p2p/wallet/jupiter/repository/tokens/db/SwapTokenEntityInserter.kt

```diff
@@ -0,0 +1,58 @@
+package org.p2p.wallet.jupiter.repository.tokens.db
+
+import androidx.collection.SimpleArrayMap
+import com.google.gson.stream.JsonReader
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.supervisorScope
+import org.p2p.wallet.jupiter.api.response.tokens.JupiterTokenResponse
+
+class SwapTokenEntityInserter(
+    private val dao: SwapTokensDao
+) {
+
+    private fun createAddressToIndexArray(routesArray: JsonReader): SimpleArrayMap<String, Int> {
+        routesArray.beginArray()
+        var index = 0
+        val addressToIndex = SimpleArrayMap<String, Int>(2000)
```

Comment:
MAP instead of LIST because get(mintAddress) for MAP is faster than looking thru all the list using indexOf